examples/mixture/plot_gmm.py

```diff
@@ -20,6 +20,9 @@
 per cluster than there are dimensions in the data, due to
 regularization properties of the inference algorithm.
 """
+
+# Modified by Thierry Guillemot <thierry.guillemot.work@gmail.com>
```

Comment:
add your name to author list when there is one or remove it. We don't keep the logs of changes in each file